bundle/metadata/annotations.yaml

```diff
@@ -6,6 +6,7 @@ annotations:
   operators.operatorframework.io.bundle.package.v1: kogito-serverless-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.25.0
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.24.1
```

Comment:
We are on 1.25.0 already.

Reply:
leftover, I generated it before updating my operator-sdk :)